src/includes/class-wordlift.php

```diff
@@ -2010,6 +2022,12 @@ private function define_public_hooks() {
 			$this->loader->add_action( 'wp_enqueue_scripts', $this->analytics_connect, 'enqueue_scripts', 10 );
 		}
 
+		// Experimentl chnges for 1496 done by Nishit
+		$rest_controller_object = $this->webhook_loader->get_rest_controller_object();
+	    $this->loader->add_action( 'wl_sync_many', array( $rest_controller_object, 'register_sync_many' ), 10, 2 );
+	    $this->loader->add_action( 'wl_sync_delete', array( $rest_controller_object, 'register_sync_delete' ), 10, 3 );
```

Comment:
You're registering the action twice, here and in the class constructor.

Reply:
Done